src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -164,7 +173,12 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 				return undefined;
 			}
 			const completionItems = Array.isArray(completions) ? completions : completions.items ?? [];
-
+			if (provider.isBuiltin) {
+				//TODO: why is this needed?
```

Comment:
Clean up?

Reply:
I want to leave that with the hope that we can figure it out eventually